context/AppContextProvider.tsx

```diff
@@ -39,13 +40,14 @@ interface AppContextProviderProps {
 
 const AppContextProvider = ({ children }: AppContextProviderProps) => {
   const t = useTranslations();
-  const [currentUser, setCurrentUser] = useState<IUser | null>(null);
+  const [currentUser, setCurrentUser] = useState<string | null>(null);
 
   const registerUser = async () => {
     const isInitiated= await Pi.initialized;
+
     if (isInitiated) {  
       try {
-        localStorage.removeItem("mapOfPiToken");
+        // localStorage.removeItem("mapOfPiToken");
```

Comment:
No need to comment lines; Please just remove.  We can always refer and/ or recover if needed through the commit history.